### ISSUE TITLE
Prediction result is not as expected.

### ISSUE THREAD

**Issue by harisgulzar1** [rishisidhu/CNN_spoken_digit#2]
Running fetch.py kept on giving following error:
`File "fetch.py", line 300, in get_wav_data
    red_data            = redgram.reshape(imheight,imwidth, 1)
ValueError: cannot reshape array of size 425 into shape (34,50,1)`

I changed the line 293:
`redgram = block_reduce(normgram, block_size = (14,13), func = np.mean)`

Now the error is removed because the redgram size is as it is required.

But the code always returns me the following result, no matter whatever digit I try to speak:
`TestFile Name:  0.wav  The Model Predicts: 5
Class  0  Confidence:  10.0
Class  1  Confidence:  10.0
Class  2  Confidence:  10.0
Class  3  Confidence:  10.0
Class  4  Confidence:  10.0
Class  5  Confidence:  10.0
Class  6  Confidence:  10.0
Class  7  Confidence:  10.0
Class  8  Confidence:  10.0
Class  9  Confidence:  10.0`

What could be the reason for that?

**Comment by Sarita67** [comment]
Have you figured this out? I'm finding the same error, it just predicts the same digit.

**Comment by harisgulzar1** [comment]
> Have you figured this out? I'm finding the same error, it just predicts the same digit.

No, it's still not solved fully. Firstly I made some changes to retrain the model correctly, but even after retraining, code doesn't give accurate results. Anyhow, training accuracy was around 97% in last epoch.
